sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -227,6 +230,22 @@ static void applyMetadata(
 
         options.setProguardUuid(
             readString(metadata, logger, PROGUARD_UUID, options.getProguardUuid()));
+
+        SdkVersion sdkInfo = options.getSdkVersion();
+        if (sdkInfo != null) {
+          final String sdkName = readString(metadata, logger, SDK_NAME, sdkInfo.getName());
+          if (sdkName != null) {
```

Comment:
I think these checks are redundant, because `sdkInfo.getName()/sdkInfo.getVersion()` are non-nullable, hence the default value is always non-nullable

Reply:
the problem is with readString, returning a nullable type - should I instead create a non-nullable return type override?

Reply:
mm, you could do that, but it's a nitpick rather - it's also fine like that. So, as you prefer :)

Reply:
added the override